index.html

```diff
@@ -178,7 +178,7 @@
                 status: "Recommendation",
                 date : "January 2015"
             },
-	    "ISO/IEC 29100" : {	
+	    "ISO/IEC-29100" : {	
```

Comment:
We can't use a slash (`/`) for reference ID. 
RegEx of reference ID is `"^[a-zA-Z0-9\\-_+\\.]+$"` (from [schema for SpecRef](https://github.com/tobie/specref/blob/master/schemas/raw-reference.json)).